pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -510,15 +587,22 @@ func (nc NodeController) tryUpdateNodeStatus(node *api.Node) (error, time.Durati
 				nc.recordNodeOfflineEvent(node)
 			}
 		}
-		_, err = nc.kubeClient.Nodes().Update(node)
-		if err != nil {
-			glog.Errorf("Error updating node %s: %v", node.Name, err)
-		} else {
-			return nil, gracePeriod, lastReadyCondition
+		if !api.Semantic.DeepEqual(readyCondition, lastReadyCondition) {
```

Comment:
Oh sorry. I didn't answer your question. LastProbeTime means the time when Kubelet posted something. ReadyCondition may be changed by the NodeController, and this is the part of code which handles timestamps for that.